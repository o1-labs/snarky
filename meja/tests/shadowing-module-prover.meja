module M = {
  type t;
};

Prover {
  module M = {
    type u;
  };
};
