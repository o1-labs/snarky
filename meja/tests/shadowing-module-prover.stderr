File "tests/shadowing-module-prover.meja", line 6, characters 9-10:
Error: There is already a module with name M.
